api/envoy/config/cluster/v3/outlier_detection.proto

```diff
@@ -19,11 +20,21 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 
 // See the :ref:`architecture overview <arch_overview_outlier_detection>` for
 // more information on outlier detection.
-// [#next-free-field: 24]
+// [#next-free-field: 25]
 message OutlierDetection {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.api.v2.cluster.OutlierDetection";
 
+  // Set of passive monitors.
+  // [#not-implemented-hide:]
+  message Monitor {
```

Comment:
I think we should use the existing [`TypedExtensionConfig`](https://github.com/envoyproxy/envoy/blob/f9aaa3f8cc8d79da57eaf559f4a8d1f22a5a359b/api/envoy/config/core/v3/extension.proto#L20) message instead of defining our own message here.

Reply:
I was not aware of `TypedExtensionConfig`. Will refactor it. Thanks!